views/route.html

```diff
@@ -0,0 +1,31 @@
+<div>
+  <kubernetes-object-describe-header resource="resource" kind="kind"></kubernetes-object-describe-header>
+  <dl class="dl-horizontal">
+    <dt>Name</dt>
+    <dd>{{resource.metadata.name}}</dd>
+    <dt>Namespace</dt>
+    <dd>{{resource.metadata.namespace}}</dd>
+    <dt>Created</dt>
+    <dd>{{resource.metadata.creationTimestamp}}</dd>
+    <dt>Host</dt>
+    <dd>{{resource.host}}</dd>
+    <dt>Path</dt>
+    <dd>{{resource.path || 'None'}}</dd>
+    <dt>Service Name</dt>
+    <dd>{{resource.serviceName}}</dd>
+  </dl>
+  <div ng-if="resource.tls">
+    <h3>TLS</h3>
+    <dl class="dl-horizontal">
+      <dt>Termination</dt>
+      <dd>{{resource.tls.termination || 'None'}}</dd>
+      <dt>Certificate</dt>
+      <dd collapse-long-text value="{{resource.tls.certificate || 'None'}}"></dd>
```

Comment:
hmm, not sure that will work with `collapseLongText`... might need to take extra whitespace out of `_collapse-long-text.html` to allow that to work